### DISCUSSION TITLE
Switch to yarn zero installs causes plugin-typescript to not find tslib

### DISCUSSION THREAD

**Discussion by paulober** [rollup/plugins#1529] [Q&A]
- Rollup Plugin Name: @rollup/plugin-typescript
- Rollup Plugin Version: 11.1.2
- Rollup Version: 3.26.0
- Operating System (or Browser):
- Node Version: v18.16.1
- Link to reproduction: just run `yarn init -2` with corepack yarn inside a npm rollup project and try to build it with rollup

### Expected Behavior
It builds normally like with npm

### Actual Behavior
```
[!] (plugin typescript) RollupError: @rollup/plugin-typescript: Could not find module 'tslib', which is required by this plugin. Is it installed
```

**Comment by burner1024** [comment]
Same with pnpm.
